Watcher/v1/summit.yaml

```diff
@@ -1,20 +1,69 @@
 rules:
 - classname: Enabled
   configs:
+  - name: ScriptQueue:1
+  - name: ScriptQueue:2
+  - name: Scheduler:1
+  - name: Scheduler:2
+  - name: ATAOS
   - name: ATDome
   - name: ATDomeTrajectory
   - name: ATHexapod
   - name: ATMCS
   - name: ATPneumatics
   - name: ATPtg
-  - name: Environment
-  - name: ATAOS
+  - name: ATSpectrograph
+  - name: ATCamera
+  - name: ATArchiver
+  - name: ATHeaderService
+  - name: MTAOS
+  - name: MTDome
+  - name: MTDomeTrajectory
+  - name: MTHexapod:1
+  - name: MTHexapod:2
+  - name: MTM1M3
+  - name: MTM2
+  - name: NewMTMount
+  - name: MTPtg
+  - name: MTRotator
+  - name: CCArchiver
+  - name: CCCamera
+  - name: CCHeaderService
+  - name: WeatherStation:1
   - name: GenericCamera
+  - name: DIMM:1
+  - name: DIMM:2
 - classname: Heartbeat
   configs:
+  - name: ScriptQueue:1
+  - name: ScriptQueue:2
+  - name: Scheduler:1
+  - name: Scheduler:2
+  - name: ATAOS
   - name: ATDome
   - name: ATDomeTrajectory
   - name: ATHexapod
-  - name: Environment
-  - name: ATAOS
+  - name: ATMCS
+  - name: ATPneumatics
+  - name: ATPtg
+  - name: ATSpectrograph
+  - name: ATCamera
+  - name: ATArchiver
+  - name: ATHeaderService
+  - name: MTAOS
+  - name: Dome
+  - name: MTDomeTrajectory
+  - name: MTHexapod:1
+  - name: MTHexapod:2
+  - name: MTM1M3
+  - name: MTM2
+  - name: NewMTMount
+  - name: MTPtg
+  - name: MTRotator
+  - name: CCArchiver
+  - name: CCCamera
+  - name: CCHeaderService
+  - name: WeatherStation:1
   - name: GenericCamera
+  - name: DIMM:1
+  - name: DIMM:2
```

Comment:
A nice improvement.

I strongly suggest alphabetizing both lists so it is easier to quickly see if a particular CSC is present. They have gotten long enough that it's tricky to see what is present and what isn't.

Reply:
OK. I alphabetized both lists. Made me realize I had missed one of the renames, so that turned out to be a very good suggestion.